### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by utkarsh2102**
Signed-off-by: Utkarsh Gupta <guptautkarsh2102@gmail.com>

This adds:
- The appropriate `SEO Tags` to `index.html`.
- Name in `CONTRIBUTORS.txt`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,24 +6,35 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>LinuxChix India</title>
-    <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Arima+Madurai">
+    <title>LinuxChix India</title>
+
+    <!-- SEO Tags -->
+    <meta name="description" content="Offcial Website of LinuxChix India">
```
Comment (shivan1b) [inline index.html]: Fix the typo here.

**Review by shivan1b — CHANGES_REQUESTED** [review]

**Comment by utkarsh2102** [comment]
@shivan1b, resolved as asked.

File: index.html [hunk 2]
```diff
@@ -6,24 +6,35 @@
...
 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Arima+Madurai">
+    <title>LinuxChix India</title>
+
+    <!-- SEO Tags -->
+    <meta name="description" content="Official Website of LinuxChix India">
+    <meta name="keywords" content="LinuxChix, LinuxChix India, LinuxChixIn, linuxchix, linuxchixin, lci, LCI">
+
+    <!-- Main CSS File -->
+    <link rel="stylesheet" href="assets/css/main.css">
```
Comment (CuriousLearner) [inline index.html:18]: `main.css` file should override the rest of the theme. Can you please fix it as it was. Categorizing bunch of imports is nice but do not change the order.

File: index.html [hunk 3]
```diff
@@ -6,24 +6,35 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>LinuxChix India</title>
-    <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Arima+Madurai">
```
Comment (CuriousLearner) [inline index.html:13]: May I know the reason for removing this font?

If it is removed, have we tried to fix the changes in `main.css` and anywhere else that it is currently being used?

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Comments inline.

File: index.html (hunk 3 above)
Comment (utkarsh2102) [reply index.html:13]: This seems to have been missed. Really sorry.
Shall I raise another PR with just the `keywords`?

Comment (CuriousLearner) [reply index.html:13]: You can do the changes here itself.

**Pull request closed without merging**